smoke-tests/smoke-e2e.bats

```diff
@@ -143,6 +143,15 @@ mk_attr() {
 
    assert_equal "$result" '"SampleRate"
 "app.metadata"
+"device.isBatteryMonitoringEnabled"
```

Comment:
note the lack of `device.batteryLevel` and `device.batteryState` here, that's because the simulator just doesn't have a battery -.-